binary-search-tree/example.rb

```diff
@@ -1,7 +1,10 @@
 class Bst
+  VERSION = 2
```

Comment:
Version 1 would suffice, prior version was 0, as it was not tracked for versions.